examples/layersColorVisible.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Globe</title>
```

Comment:
`title`  is wrong. 
Maybe we should start adding a short comment at the top of our examples, to explain what they are about?

(also please fix indentation)